### PR TITLE
Add new msg layout

### PR DESCRIPTION
**Pull request by cfpwastaken**

### CODE REVIEW

File: styles/style.css [hunk 1]
```diff
@@ -200,11 +200,14 @@ textarea {
 .message-author-name {
 	font-weight: bold;
 	font-family: Consolas, monospace, Arial;
+	margin-top: -45px;
+	margin-left: 70px;
```
Comment (j0code) [inline styles/style.css:204]:
```suggestion
	display: flex;
	align-self: center;
	padding-left: 10px;
```

File: styles/style.css [hunk 2]
```diff
@@ -200,11 +200,14 @@ textarea {
...
 }
 
 .message-content {
 	overflow-wrap: anywhere;
 	padding: 2px;
+	margin-top: 30px;
```
Comment (j0code) [inline styles/style.css:210]:
```suggestion
	display: flex;
```

**Review by j0code — CHANGES_REQUESTED** [review]
this is more dynamic :u

**Comment by cfpwastaken** [comment]
:o you finally reviewed

**Pull request merged**